packages/hardhat/contracts/ToKanBan.sol

```diff
@@ -3,21 +3,35 @@
 pragma solidity >=0.7.0 <=0.8.0;
 
 import "@openzeppelin/contracts/utils/Counters.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
 
-contract ToKanBan{
+contract ToKanBan is ReentrancyGuard{
     //Events
     event taskSubmitted(uint task_id, uint funds, string detail);
     event taskRequested(uint task_id, address raider, uint requestId);
     event assigned(uint  task_id,address raiderApproved);
     event taskForReviewed(uint task_id);
     event taskReviewRevoke(uint task_id);
     event taskCompleted(uint task_id, uint fundReleased);
-
+    
```

Comment:
@rpal3058 You might want to add an automatic formatter to your local editor - this line is just empty with some whitespace in it.

Reply:
@sazerac-eth thanks. Ya I have meaning to add in prettier